src/main/java/org/nuxeo/ecm/platform/groups/audit/service/acl/AclExcelLayoutBuilderMultiColumn.java

```diff
@@ -113,36 +115,36 @@ protected void renderInit() {
         acceptStyle = excel.newColoredCellStyle(ByteColor.GREEN);
 
         acceptStyleLeft = excel.newColoredCellStyle(ByteColor.GREEN);
-        acceptStyleLeft.setBorderLeft(CellStyle.BORDER_THIN);
+        acceptStyleLeft.setBorderLeft(BorderStyle.THIN);
         acceptStyleLeft.setLeftBorderColor(excel.getColor(ByteColor.BLACK).getIndex());
 
         acceptStyleRight = excel.newColoredCellStyle(ByteColor.GREEN);
-        acceptStyleRight.setBorderRight(CellStyle.BORDER_THIN);
+        acceptStyleRight.setBorderRight(BorderStyle.THIN);
         acceptStyleRight.setRightBorderColor(excel.getColor(ByteColor.BLACK).getIndex());
 
         denyStyle = excel.newColoredCellStyle(ByteColor.RED);
-        denyStyle.setFillPattern(CellStyle.THIN_FORWARD_DIAG); // TODO:
-                                                               // generaliser
-                                                               // autres
-                                                               // cellules
+        denyStyle.setFillPattern(FillPatternType.THIN_FORWARD_DIAG);  // TODO:
+                                                                      // generaliser
+                                                                      // autres
+                                                                      // cellules
```

Comment:
As you're changing this line, rewrite the comment in English (generalize to other cells) and one one line.